Source/Upstream: Yes, in v238-stable branch

From fc50fa43e7bc93bef84dd6ac9adf7389ce370531 Mon Sep 17 00:00:00 2001
From: futpib <futpib@gmail.com>
Date: Fri, 16 Mar 2018 17:25:14 +0300
Subject: [PATCH 22/51] hwdb: fix comment suggested `udevadm trigger` command
 (#8465)

(cherry picked from commit d24e70fe8b6bebd9abe4c1578d22cb5536044cc2)
---
 hwdb/60-sensor.hwdb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hwdb/60-sensor.hwdb b/hwdb/60-sensor.hwdb
index 2c8c4467e..616c39013 100644
--- a/hwdb/60-sensor.hwdb
+++ b/hwdb/60-sensor.hwdb
@@ -15,7 +15,7 @@
 #   /etc/udev/hwdb.d/61-sensor-local.hwdb
 # and add your rules there. To load the new rules execute (as root):
 #   systemd-hwdb update
-#   udevadm trigger -y `dirname $(udevadm info -n "/dev/iio:deviceXXX" -q path)`
+#   udevadm trigger -v -p DEVNAME=/dev/iio:deviceXXX
 # where /dev/iio:deviceXXX is the device in question.
 #
 # If your changes are generally applicable, preferably send them as a pull
-- 
2.16.3

